Sources/Swift/Integrations/UserFeedback/SentryUserFeedbackWidgetButtonMegaphoneIconView.swift

```diff
@@ -0,0 +1,174 @@
+import UIKit
+
+class SentryUserFeedbackWidgetButtonMegaphoneIconView: UIView {
+    init(config: SentryUserFeedbackConfiguration) {
+        super.init(frame: .zero)
+    
+        let svgLayer = CAShapeLayer()
+        svgLayer.path = megaphoneShape
+        
+        func configureLightTheme() {
+            svgLayer.fillColor = config.theme.foreground.cgColor
+        }
+        if #available(iOS 12.0, *) {
+            if UIScreen.main.traitCollection.userInterfaceStyle == .dark {
+                svgLayer.fillColor = config.darkTheme.foreground.cgColor
+            } else {
+                configureLightTheme()
+            }
+        } else {
+            configureLightTheme()
+        }
+        
+        svgLayer.fillRule = .evenOdd
```

Comment:
l: This is just a suggestion to use alongside the megaphoneShape suggestion.
Feel free to ignore it 

```suggestion
        svgLayer.path = megaphoneShape
        svgLayer.lineWidth = 1.5
        svgLayer.lineJoin = .round
        svgLayer.fillColor = UIColor.clear.cgColor
        
        func configureLightTheme() {
            svgLayer.strokeColor = config.theme.foreground.cgColor
        }
        if #available(iOS 12.0, *) {
            if UIScreen.main.traitCollection.userInterfaceStyle == .dark {
                svgLayer.fillColor = config.darkTheme.foreground.cgColor
            } else {
                configureLightTheme()
            }
        } else {
            configureLightTheme()
        }
        
```